decidim-budgets/app/helpers/decidim/budgets/projects_helper.rb

```diff
@@ -80,6 +83,45 @@ def current_rule_description
           t(".vote_threshold_percent_rule.description", minimum_budget: budget_to_currency(current_order.minimum_budget))
         end
       end
+
+      # Serialize a collection of geocoded projects to be used by the dynamic map component
+      #
+      # geocoded_projects - A collection of geocoded projects
+      def projects_data_for_map(geocoded_projects)
+        geocoded_projects.map do |project|
+          project_data_for_map(project)
+        end
+      end
+
+      def project_data_for_map(project)
+        project
+          .slice(:latitude, :longitude, :address)
+          .merge(
+            title: decidim_html_escape(translated_attribute(project.title)),
+            description: html_truncate(decidim_sanitize_editor(translated_attribute(project.description)), length: 100),
+            icon: icon("projects", width: 40, height: 70, remove_icon_class: true),
+            link: ::Decidim::ResourceLocatorPresenter.new([project.budget, project]).path
+          )
+      end
+
+      def project_preview_data_for_map(project)
```

Comment:
Where are we using the `project_preview_data_for_map` method?

I could not find any references to this method elsewhere than in this file and in the spec.